dotnet/frontend/Controllers/GreetingController.cs

```diff
@@ -48,27 +48,31 @@ public async Task<string> GetAsync()
         {
             using var span = _tracer.StartActiveSpan("Preparing Greeting");
             {
-
                 span.SetAttribute("testAttribute", "Greeting");
                 Baggage.Current.SetBaggage("testBaggage", "Greetings");
                 var httpClient = _clientFactory.CreateClient();
                 var name = await GetNameAsync(httpClient);
-                var message = await GetMessage(httpClient);
+                var message = await GetMessageAsync(httpClient);
 
-                return $"Hello {name}, {message}";
+                using var render_span = _tracer.StartActiveSpan("🎨 render greeting ✨");

```

Comment:
.NET prefers camelCase for local variables.

```suggestion
                using var renderSpan = _tracer.StartActiveSpan("🎨 render greeting ✨");
```

Reply:
Aaaa! My Ruby's showing.